Makefile

```diff
@@ -59,18 +59,21 @@ ci:
 	$(MAKE)
 	$(MAKE) install
 	$(MAKE) -C test all
+	$(MAKE) -C pkg tag
```

Comment:
probably not, as then the tags will exist for the builds (although where there are signature checks it wont matter). So we won't catch missing tags.

But its unclear to me if we run with a fresh docker engine for each build so this may still affect the next build.